src/apps/loan-list/materials/material-decorator.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+import {
+  GetMaterialManifestationQuery,
+  useGetMaterialManifestationQuery
+} from "../../../core/dbc-gateway/generated/graphql";
+import SelectableMaterial from "./selectable-material";
+import StackableMaterial from "./stackable-material";
+import { useText } from "../../../core/utils/text";
+
+interface MaterialDecoratorProps {
+  materialType: string;
+  faust: string;
```

Comment:
We have a faust type here: https://github.com/reload/dpl-react/blob/feature/dpl-react-disclosure-DDFSOEG-168/src/core/utils/types/ids.ts#L1